openpype/hosts/flame/api/lib.py

```diff
@@ -334,3 +318,247 @@ def process(self, project_name):
 
     policy_wiretap = GetProjectColorPolicy(_log=_log)
     return policy_wiretap.process(project_name)
+
+
+def get_segment_data_marker(segment, with_marker=None):
+    """
+    Get openpype track item tag created by creator or loader plugin.
+
+    Attributes:
+        segment (flame.PySegment): flame api object
+        with_marker (bool)[optional]: if true it will return also marker object
+
+    Returns:
+        dict: openpype tag data
+
+    Returns(with_marker=True):
+        flame.PyMarker, dict
+    """
+    for marker in segment.markers:
+        comment = marker.comment.get_value()
+        color = marker.colour.get_value()
+        name = marker.name.get_value()
+
+        if (name == MARKER_NAME) and (
+                color == COLOR_MAP[MARKER_COLOR]):
+            if not with_marker:
+                return json.loads(comment)
+            else:
+                return marker, json.loads(comment)
+
+
+def set_segment_data_marker(segment, data=None):
```

Comment:
This function will always return `True` which does not make much sense...

Reply:
true